relint/__main__.py

```diff
@@ -42,6 +42,14 @@ def parse_args(args):
     parser.add_argument(
         "-W", "--fail-warnings", action="store_true", help="Fail for warnings."
     )
+    parser.add_argument(
+        "--msg-template",
+        metavar="MSG_TEMPLATE",
+        type=str,
+        default="{filename}:{line_no} {test.name}\nHint: {test.hint}\n{match}",
+        help="Template used to display messages. "
+        "Default: {filename}:{line_no} {test.name}\nHint: {test.hint}\n{match}",
```

Comment:
`argparse` remove newlines, maybe display the `\n` in the help text.

```suggestion
        r"Default: {filename}:{line_no} {test.name}\nHint: {test.hint}\n{match}",
```